src/rcx/src/extFlow.cpp

```diff
@@ -318,6 +318,7 @@ uint extMain::initSearchForNets(int* X1,
   }
   const uint layerCnt = n + 1;
 
+  delete _search;
   _search = new GridTable(&maxRect, 2, layerCnt, pitchTable, X1, Y1);
```

Comment:
If you are comfortable, this is a case where std::unique_ptr and make_unique would be a more elegant solution.  If not I can merge it as is.